# Import the data into memory

In [1]:
import pandas as pd 

filepath = 'bin/20240224-composer-raw.csv'
df = pd.read_csv(filepath)

In [3]:
df[['symphony_name', 'symphony_id', 'annualized_return', 'max_drawdown']].sort_values(by = ['annualized_return', 'max_drawdown'])

symphony_name           symphony_id  \
49                     Excellent Adventures (with NVDA)  0Aj9HRgodIFOZpIPEql0   
0     V 1.0 | ?? Beta Baller | Deez | 6MO: AR 13740....  N7nxAmvVwxT7kLkaoTLs   
1                                                  VIXY  KUVweL39UU0X3d5lOs8q   
2            [DEV] ?? Beta Baller v0.21 - $Deez | OCT22  uOG8uEJzZdfZLKc4RJCe   
3                                               Yes 2.0  CZ7QAYvhRIlmwJ5VXlGj   
...                                                 ...                   ...   
1073                W: V1a BWC: Filtered BTC Strategies  Hiw0UVAh07Waqjc9WrnE   
1154                               What Have I Done. v3  AKQ4eDb3cH2dLkhiAQQf   
1097                    V2a What Have I Done - K-1 Free  7vb59fKKEWhGQQalgBhp   
229                 CQ | BTC ₿ Rally Rider v4 | 1.12.24  ENl5lR8xIcCRdTrJ7Uki   
208                                       Bitcoin bebop  vwAGDxbm2FxAr80Q7n3K   

      annualized_return  max_drawdown  
49              -0.5873        0.2203  
0               -0.5037        1.0000  
1               -0.3598        0.9974  
2               -0.3549        0.9997  
3               -0.2883        0.9932  
...                 ...           ...  
1073           143.5660        0.1323  
1154           156.4600        0.4678  
1097           174.8168        0.4419  
229            284.6349        0.0559  
208           9946.2149        0.0569  

[2594 rows x 4 columns]

# Query algo based on the following criteria
1. backtest_start_date before 2012
2. Max drawdown less than 21%
3. Annual return > 25%
4. Sort by annual return desc

In [4]:
# df.query('backtest_start_date < 2012-01-01')
df['backtest_start_date'] = pd.to_datetime(df['backtest_start_date'])

In [5]:
pd.set_option('display.max_colwidth', None)
df[
    (df['backtest_start_date'] < '2012-01-01') 
    & (df['max_drawdown'] < .21)
    & (df['annualized_return'] > .25)
][
    ['symphony_id','max_drawdown', 'annualized_return', 'symphony_name', 'created_by', 'backtest_start_date', 'returns_1m', 'returns_3m']
].sort_values(by = ['annualized_return', 'max_drawdown'], ascending=False)

symphony_id  max_drawdown  annualized_return  \
2269  SUEKwZUVBazsaWRZqXOU        0.2044             1.1045   
2238  rIQLs61vYvuevCVG0x2q        0.1816             1.0268   
2226  fjRmx3nhsr7cDnbZPbp2        0.1909             0.9923   
2207  m5jN3RYFAXyuOAD15wmg        0.1426             0.9574   
2111  STbTdDOmWhSYR5wUGJ71        0.1926             0.7759   
2023  glzUK5BYkErFnMXBrXCb        0.1979             0.6772   
1869  pAG3NN3phzNhHz1ESrku        0.0792             0.5428   
1867  4Ed4UzYvB0wAmA6cKJcX        0.1649             0.5363   
1850  2gX2Ch9EsG5MmJ9yjAvN        0.1980             0.5215   
1842  yPNTAMuDZVcVCBRYAG2q        0.1649             0.5161   
1781  ihmip9A197RX7YApiuv2        0.1425             0.4820   
1783  8GXwyfYVrLyHAD20wbGr        0.1628             0.4819   
1782  GJDRaKZG0U1PoysGaSs8        0.1672             0.4816   
1913  nkPYUV4Ksf1FMD3cXwOG        0.1622             0.4527   
1712  kMpXOGPIEMSmLRJrxDW9        0.1667             0.4502   
1713  GvGfeBTZayKTVtvo5joH        0.1329             0.4490   
1685  sPmonUWG0WYWNUoJB7lk        0.1984             0.4353   
1630  fTYgyX246K6eXVNHJ1Ol        0.1849             0.4079   
1622  CklMKxP88yXM9Lid0iAa        0.1439             0.4051   
1580  oMSC6CujvudCMYfadSRy        0.1611             0.3869   
1557  7gmq0oyEzXfodZZl9U99        0.1524             0.3723   
1539  JUly3gwBeFwVLt1Dm3su        0.1391             0.3686   
1523  O87Ik98lWKFZLc5hW4yE        0.1630             0.3584   
1506  k69jo6p0ZMhitcoPbOQl        0.1806             0.3554   
1667  Y9INEoYF2uZbGpt8rjt9        0.1685             0.3476   
1495  G7TZ49Ogj9cs2CF9eQK7        0.1531             0.3459   
1471  L4a66U6kU8rcqdN2C8bG        0.0990             0.3411   
1468  SsUbAgdtxZHN2c7MKdpQ        0.1618             0.3369   
1546  I26x5HG96mhirzjUbC9u        0.1677             0.3215   
1351  Kfd8Hp3FcLWu4UIFPXig        0.1281             0.3014   
1329  ndFy0AuwtBKYrkEszajp        0.1413             0.2973   
1323  mJKMWMDYGsQnkQBm1dEN        0.1400             0.2958   
1293  Mw6D89S4J50tpkY6U95e        0.1806             0.2890   
1363  Vb1e9MWwEpwuNVXDlZXw        0.1849             0.2865   
1409  OuOTwhROHLv2iwVF2COU        0.1956             0.2837   
1662  eZxZqmCgFYwjULSOtqJG        0.1885             0.2835   
1259  s4bcd3bkL0DCzJeV8sYS        0.1305             0.2797   
1230  0ZmKN2022XnilEaT1lCh        0.1475             0.2715   
1566  YFo7FeVJkHmUTWnXUNS0        0.1955             0.2656   
1189  4D2CLm1qWZaPLXhSHXgW        0.1559             0.2587   
1175  CEMVa82TFeCXYTnlSP5l        0.1128             0.2549   
1318  dUoV5I1iW59krj8yqDf8        0.1905             0.2548   
1169  qfNzXsjnvXTzQyJ7h35U        0.1806             0.2533   

                                                                                                symphony_name  \
2269  REINVEST [12/30/22 Invested] [BETA] V4.1.2 | A Better "LETF minimum drawdown" | DJKeyhole 🧙‍♂️ | Apollo   
2238                                                                      🐟 Swedish Fish v1.a | NoWAM | 50/50   
2226                                                                                               (T)QQQ/PSQ   
2207                       Copy of [BETA] V4.1.1 | A Better "LETF minimum drawdown" | DJKeyhole 🧙‍♂️ | No UGE   
2111                                                                       🧸 Gummy Bears v1.0 | NoWAM | 60/40   
2023                                                                                     (T)QQQ/PSQ - No SQQQ   
1869                                                         V1.0 Smooth Criminal | BWC + Anansi | 2011-11-15   
1867                                                                           BB Framework with only QQQ/PSQ   
1850                                                                                    OG Boring TQQQ Trendz   
1842                                                                                    QQQ/PSQ - No Leverage   

# Search Algo based on the name

In [7]:
df.query('symphony_name.str.contains("Sandy\'s Golden Dragon")')[
    ['symphony_name', 'created_by', 'symphony_id','max_drawdown', 'annualized_return', 'backtest_start_date', 'returns_1m', 'returns_3m', 'Treynor Ratio']
].sort_values(by = ['Treynor Ratio', 'annualized_return', 'max_drawdown'], ascending=False)

symphony_name  \
1508                                               Sandy's Golden Dragon V4.3a (vol hedge?)   
1472                              Sandy's Golden Dragon V4.6a (vol hedge? + beta hedge 2.0)   
1474                                  Sandy's Golden Dragon V4.5a (vol hedge? + beta hedge)   
1456                      Sandy's Golden Dragon V4.7a (vol hedge? + beta hedge 2.0 + BAB 2)   
1479                                               Sandy's Golden Dragon V4.4a (vol hedge?)   
1421                               Sandy's Golden Dragon V4.6 (vol hedge? + beta hedge 2.0)   
1447                                                Sandy's Golden Dragon V4.3 (vol hedge?)   
1443                                                Sandy's Golden Dragon V4.4 (vol hedge?)   
1415                                   Sandy's Golden Dragon V4.5 (vol hedge? + beta hedge)   
1405                       Sandy's Golden Dragon V4.7 (vol hedge? + beta hedge 2.0 + BAB 2)   
915                                                                   Sandy's Golden Dragon   
1329                                                      Copy of Sandy's Golden Dragon 3.0   
1323                                                              Sandy's Golden Dragon 3.0   
1307  Sandy's Golden Dragon V5 - Replace TQQQ FTLT w/ Base 2 - FTLT or Bull or Bonds (UVXY)   
1286                                                  Sandy's Golden Dragon V5 (alpha copy)   
1283                                                             Sandy's Golden Dragon V4.1   
1252                                                             Sandy's Golden Dragon V4.0   
735                                                              Sandy's Golden Dragon V2.0   
1099                                                 V1a 🦴 Sandy's Golden Dragon Skeleton 🦴   
1063                                                     🦴 Sandy's Golden Dragon Skeleton 🦴   
1004  [Long Backtest Copy] Sandy's Golden Dragon V4.7 (vol hedge? + beta hedge 2.0 + BAB 2)   
685                                      [No LETF] Sandy's Golden Dragon V4.4a (vol hedge?)   
674                                       [No LETF] Sandy's Golden Dragon V4.4 (vol hedge?)   
353                                                 NO LETF V of Sandy's Golden Dragon V2.0   

           created_by           symphony_id  max_drawdown  annualized_return  \
1508   Ronniem90#9038  EziYIiWyUrHgSbgNACKW        0.1557             0.4447   
1472   Ronniem90#9038  r6GvWpanFtK0dzcij45C        0.1411             0.4337   
1474   Ronniem90#9038  Pfe4QDxrbEehr2s1ATJ5        0.1408             0.4345   
1456   Ronniem90#9038  xxFhi7WXusETROpYpFwY        0.1362             0.4256   
1479   Ronniem90#9038  BBjFrVKLQ0v1C1LSDs8C        0.1508             0.4352   
1421  sandypants#7720  O7isfqiMIhj9yh7TCz5v        0.1454             0.4134   
1447  sandypants#7720  oQ6HRwjc7frO7SReuh1T        0.1610             0.4180   
1443  sandypants#7720  uXpJTnFfgsrZ78wtPfAi        0.1537             0.4195   
1415  sandypants#7720  BiuErSY95m0KZIHhEpiI        0.1434             0.4117   
1405  sandypants#7720  a41bkkjlaZrUCMZ6FECG        0.1379             0.4057   
915   sandypants#7720  NKRwNRCR6unqJX9rEArH        0.0759             0.3824   
1329  sandypants#7720  ndFy0AuwtBKYrkEszajp        0.1413             0.2973   
1323  sandypants#7720  mJKMWMDYGsQnkQBm1dEN        0.1400             0.2958   
1307   Ronniem90#9038  kC8l3AaTph8SpSatG6SV        0.1099             0.3682   
1286  sandypants#7720  8yIGEaCoJHSc9ySog8D4        0.1147             0.3617   
1283  sandypants#7720  tcCMzHo0R3h9TvZkVU2L        0.1371             0.3575   
1252  sandypants#7720  0NKvTLaGhMufWWlMPyqS        0.1383             0.3451   
735   sandypants#7720  A8cSaBkkiigN7h3PGXcr        0.0744             0.4559   
1099   Ronniem90#9038  4Kwp6YkxmnlNkIC1PHXM        0.1630             0.2984   
1063  sandypants#7720  tmjjrLLhKeCRhGIqMROP        0.1574             0.2869   
1004  sandypants#7720 

# 2023-09-04 Investments
1. **OG Boring TQQQ Trendz** | 2gX2Ch9EsG5MmJ9yjAvN | 2011-10-19
2. **Blend: Regime Switching** | 8GXwyfYVrLyHAD20wbGr | 2011-10-19
3. **Safe Dips/Rips l 10 Jan 2011** | kMpXOGPIEMSmLRJrxDW9 | 2011-10-19
4. **Volatility Index Black Swan Catcher W/TINA (Only meant for bear market)** | fTYgyX246K6eXVNHJ1Ol | 2011-10-05	
5. **Sandy's Golden Dragon 3.0** | mJKMWMDYGsQnkQBm1dEN | 2011-10-21
6. **Simple Beat v1** | x9yVHOo1H4Q6qybaaBUW | 2011-10-21 | 	26.6%
7. **Idk Part Too** | U6IUQXiYMgrlGqq2IqOJ | 2014-01-21		
8. **Simple long term FTLT l 12 Jun 2014** | g95Baj0E4q6ygmJ7Z0tU | 2014-06-13	
9. **Buy the Dips Else Not Boring v1.2** | VmMTI2FUrG1gnIq5xsOo | 2011-12-08
10. **V2 QLD For The Long Term** | Garen Mod | RJPNmSmfFkgGv74sfAFx | 2008-05-16
11. **2x Leveraged | Dereck's Secular Market Framework** | QzLDGjjZKlztt8Ozelhj | 2007-04-11

# 2023-09-09 Investments

### **BB Framework with only QQQ/PSQ** | belcampo69 | 4Ed4UzYvB0wAmA6cKJcX | 2011-10-19
- Better QQQ
- Long and short Nasdaq 100
- Threashold: Daily

### **OG Boring TQQQ Trendz** | sandypants | 2gX2Ch9EsG5MmJ9yjAvN | 2011-10-19
- Not boring dip buyer (30%), Trend (35%), 2X Tech with Black swan catcher (12.5%), HFEA (22.5%)
- Trend is consist of equity, treasury, gold, dollar, commodity
- HFEA is stands for Hedgefund Excellent Adventure. It handles normal market conditions, rising and falling interest rate.
- Multiple strategy combined, such as umbrella, HFEA, Black swan catcher, dip buyer. Complicated
- Half risk of QQQ. 3x the returns in long term.
- Threshold: 5%

### **Blend: Regime Switching** | Michael B | 8GXwyfYVrLyHAD20wbGr | 2011-10-19
- Simple Regime Switch (50%) and Hedge Regime Switch (50%)
- Simple Regime Switch is consist of 2X QQQ or Fund Surf, and Dollar/Treasury/Consumer hedge.
- Hedge Regime Switch is consist of black swan catcher, Fund Surf, short and long strategy.
- Complicated with mix strategy. Little diversification.
- Small amount of trading expected.
- Beats SPY/QQQ in long term. Less risk.
- Threshold: 25%

### **Safe Dips/Rips l 10 Jan 2011** | BrianE | kMpXOGPIEMSmLRJrxDW9 | 2011-10-19
- Leverage Tech/SPY, and short volatility.
- Only Beats SPY/QQQ in long term. Acceptable result. Less risk than equity.
- Relatively simple to understand. Similar to Pop Bots.
- Threshold: 10%

### **Volatility Index Black Swan Catcher W/TINA (Only meant for bear market)** | Cashen | fTYgyX246K6eXVNHJ1Ol | 2011-10-05	
- Advance Pop Bots with Black Swan catcher.
- Leverage QQQ/SPY/SMH involved.
- Beats SPY/QQQ in long term. Acceptable result. Less risk than equity.
- Threshold: 10%

### **The Sandy Dragon** | sandypants | gwwdpdwmU2P6Z4YJ3Y7T | 2017-05-25	
- Less than 10 years backtest
- Complicated
- Diversification. Volatility (21%), Momentum (18%), Gold (19%), Growth/Tech (24%), Interest Rate (18%)
- Beats SPY/QQQ in long term. Less risk. Excellent calmer ratio.
- Threashold: 2%

### **QLD For The Long Term (62% APY 38.7% DD) Since Apr 2007** | dereck1776 | q1iJqyUqHDdGFl6H7ZtT | 2007-02-16
- Leverage 2x QQQ, with short/treasury/technology
- Use momentum and handle sideway market in QQQ
- Beats SPY/QQQ in short/long term.
- Similar risk to QQQ.
- Tremendous result. 3 digit % annual return.
- Threashold: Daily

### **2x Leveraged | Dereck's Secular Market Framework** | dereck1776 | QzLDGjjZKlztt8Ozelhj | 2007-04-11
- Secular Bull and Bear handling.
- 2X leverage on Stock/Bonds/Technology
- Use short and consumer stable for bear market.
- Simple strategy.
- Beats SPY/QQQ long term. Less risk.
- Threashold: 5%

### **Buy the Dips Else Not Boring v1.2** | mwc360 | VmMTI2FUrG1gnIq5xsOo | 2011-12-08
- Bull: Leverage Tech
- Bear: Leverage Financial/Dollar/Gold/Treasury/VIX
- Not boring with mean reversal.
- Beats SPY all the time. Beats QQQ some time.
- Good quality.
- Threashold: 5%

### **Simple long term FTLT l 12 Jun 2014** | BrianE#5796 | g95Baj0E4q6ygmJ7Z0tU | 2014-06-13
- Bull: Leverage Tech/Energy/Treasury/Vix
- Bear: Dividend ETFs, Consumer stable
- Simple. Consist of FTLT and Hedge only
- Beats SPY/QQQ in short/long term. Less risk.
- Good quality.
- Threashold: 5%

### **Idk Part Too** | sandypants | U6IUQXiYMgrlGqq2IqOJ | 2014-01-21
- A bit too complicated.
- The best from Sandy.
- Diversification, similar to Dragon. But on Equity (40%), Long Volatility (12.5%), Bonds (27.5%), Trend (20%)
- Beats SPY/QQQ in long term. Less risk. Excellent calmer ratio.
- Threashold: 2%

In [8]:
symphony_list = [
    '2gX2Ch9EsG5MmJ9yjAvN', 
    '8GXwyfYVrLyHAD20wbGr', 
    'kMpXOGPIEMSmLRJrxDW9', 
    'fTYgyX246K6eXVNHJ1Ol', 
    'gwwdpdwmU2P6Z4YJ3Y7T',
    'q1iJqyUqHDdGFl6H7ZtT', 
    'QzLDGjjZKlztt8Ozelhj', 
    'VmMTI2FUrG1gnIq5xsOo',
    'g95Baj0E4q6ygmJ7Z0tU', 
    'U6IUQXiYMgrlGqq2IqOJ',
    '4Ed4UzYvB0wAmA6cKJcX'
]

df_top = df[df['symphony_id'].isin(symphony_list)]

In [9]:

# cols = [
#     'symphony_id',
#     'Treynor Ratio',
#     'max_drawdown',
#     'annualized_return',
#     'symphony_name',
#     'created_by',
#     'backtest_start_date',
#     'returns_1m',
#     'returns_3m',
#     'Beta',
#     'sharpe_ratio',
#     'calmar_ratio',
#     'last_day_holdings'
# ]

cols = [
    'symphony_id',
    'Treynor Ratio',
    'symphony_name',
    'annualized_return',
    'max_drawdown'
]
df_top[cols].sort_values(by = ['Treynor Ratio', 'annualized_return', 'max_drawdown'], ascending=False)


symphony_id  Treynor Ratio  \
2206  q1iJqyUqHDdGFl6H7ZtT        46.7045   
1712  kMpXOGPIEMSmLRJrxDW9         5.3063   
1371  gwwdpdwmU2P6Z4YJ3Y7T         3.5695   
1867  4Ed4UzYvB0wAmA6cKJcX         3.5602   
1850  2gX2Ch9EsG5MmJ9yjAvN         2.6840   
1783  8GXwyfYVrLyHAD20wbGr         2.3171   
1592  U6IUQXiYMgrlGqq2IqOJ         1.7732   
1630  fTYgyX246K6eXVNHJ1Ol         1.4663   
1682  QzLDGjjZKlztt8Ozelhj         1.0570   
1703  g95Baj0E4q6ygmJ7Z0tU         0.8442   
1638  VmMTI2FUrG1gnIq5xsOo         0.6345   

                                                                symphony_name  \
2206                  QLD For The Long Term (62% APY 38.7% DD) Since Apr 2007   
1712                                             Safe Dips/Rips l 10 Jan 2011   
1371                                                         The Sandy Dragon   
1867                                           BB Framework with only QQQ/PSQ   
1850                                                    OG Boring TQQQ Trendz   
1783                                                  Blend: Regime Switching   
1592                                                             Idk Part Too   
1630  Volatility Index Black Swan Catcher W/TINA (Only meant for bear market)   
1682                         2x Leveraged | Dereck's Secular Market Framework   
1703                                      Simple long term FTLT l 12 Jun 2014   
1638                                        Buy the Dips Else Not Boring v1.2   

      annualized_return  max_drawdown  
2206             0.6272        0.3873  
1712             0.4502        0.1667  
1371             0.6397        0.1339  
1867             0.5363        0.1649  
1850             0.5215        0.1980  
1783             0.4819        0.1628  
1592             0.5009        0.1256  
1630             0.4079        0.1849  
1682             0.3031        0.3710  
1703             0.5996        0.2721  
1638             0.4185        0.2845

In [ ]:
import json

tickers = []
for item in df_top['last_day_holdings']:
    data = json.loads(item.replace("'", '"'))
    # temp = list(data.keys())
    for key in data.keys():
        tickers.append(key)
set(tickers)

In [15]:
df[
    (df['backtest_start_date'] < '2012-01-01') 
    & (df['max_drawdown'] < .21)
    & (df['annualized_return'] > .25)
][
    ['symphony_id','max_drawdown', 'annualized_return', 'Treynor Ratio','symphony_name', 'created_by', 'backtest_start_date', 'returns_1m', 'returns_3m']
].sort_values(by = ['Treynor Ratio', 'annualized_return', 'max_drawdown'], ascending=False).head(15)

symphony_id  max_drawdown  annualized_return  Treynor Ratio  \
1580  SUEKwZUVBazsaWRZqXOU        0.2045             1.1182       208.5334   
1541  m5jN3RYFAXyuOAD15wmg        0.1426             0.9712        86.8089   
1264  ihmip9A197RX7YApiuv2        0.1425             0.4879         5.4157   
1216  kMpXOGPIEMSmLRJrxDW9        0.1667             0.4537         4.5453   
1247  GJDRaKZG0U1PoysGaSs8        0.1673             0.4738         3.7916   
1072  JUly3gwBeFwVLt1Dm3su        0.1391             0.3642         3.1204   
1137  CklMKxP88yXM9Lid0iAa        0.1439             0.4013         2.8572   
1295  4Ed4UzYvB0wAmA6cKJcX        0.1573             0.5137         2.4810   
1303  2gX2Ch9EsG5MmJ9yjAvN        0.1980             0.5201         2.2416   
1069  G7TZ49Ogj9cs2CF9eQK7        0.1531             0.3585         1.9961   
1222  8GXwyfYVrLyHAD20wbGr        0.1628             0.4573         1.5533   
1183  eZxZqmCgFYwjULSOtqJG        0.1885             0.2830         1.4845   
1147  fTYgyX246K6eXVNHJ1Ol        0.1849             0.4114         1.3126   
1167  sPmonUWG0WYWNUoJB7lk        0.1984             0.4200         1.1936   
1070  YFo7FeVJkHmUTWnXUNS0        0.1781             0.2514         1.1873   

                                                                                                symphony_name  \
1580  REINVEST [12/30/22 Invested] [BETA] V4.1.2 | A Better "LETF minimum drawdown" | DJKeyhole 🧙‍♂️ | Apollo   
1541                       Copy of [BETA] V4.1.1 | A Better "LETF minimum drawdown" | DJKeyhole 🧙‍♂️ | No UGE   
1264                  20% 3.0.4a + 80% SHY 10Y AR 59.7% DD 11.7% Beta 0.17 | 4NOV12-4NOV22| Tribute to Pareto   
1216                                                                             Safe Dips/Rips l 10 Jan 2011   
1247                                                                               Copy of Boring TQQQ Trendz   
1072                                       Single Pops (Medium Leverage - No Black Swan Catcher) l 2 Nov 2011   
1137                                                                                                 Vixation   
1295                                                                           BB Framework with only QQQ/PSQ   
1303                                                                                    OG Boring TQQQ Trendz   
1069                                      V1 Barry Bonds | DereckN | Beta: 0.16 | Sharpe: 2.06 | Calmar: 2.65   
1222                                                                                  Blend: Regime Switching   
1183                                                          Bond Baller | 29% APY 18.8% Max DD | Jan 1 2007   
1147                                  Volatility Index Black Swan Catcher W/TINA (Only meant for bear market)   
1167                                                                     Overfit PoS (lol mod) l 26 July 2022   
1070                                                              A Different SPY 25.4%/19.1% DD May 4th 2007   

               created_by backtest_start_date  returns_1m  returns_3m  
1580       DJKeyhole#8484          2011-10-19      0.0276      0.1309  
1541  optimal-energy#8838          2011-10-12      0.0477      0.1005  
1264      belcampo69#6180          2011-10-24      0.0276      0.0814  
1216          BrianE#5796          2011-10-19      0.0257      0.1235  
1247      sandypants#7720          2011-10-19     -0.0727     -0.0790  
1072          BrianE#5796          2011-10-19      0.0225      0.1060  
1137            .briane#0          2011-10-19      0.0204      0.0934  
1295      belcampo69#6180          2011-10-19      0.0603      0.1435  
1303      sandypants#7720          2011-10-19     -0.0704     -0.0914  
1069      dereck1776#5687          2011-09-14     -0.0029     -0.0377  
1222       Michael B#1108          2011-10-19     -0.0720     -0.0217  
1183         dereck1776#0          2006-07-21     -0.0238     -0.0511  
1147          Cashen#